#!/usr/bin/env python
PACKAGE = "dacbot_controller"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#gen.add("w1_w1_param", double_t, 0, "Weight neuron 1 to 1", 1.4, -4,   4)
#gen.add("w1_w2_param", double_t, 0, "Weight neuron 1 to 2", 0.7, -4,   4)
#gen.add("w2_w1_param", double_t, 0, "Weight neuron 2 to 1", -0.7, -4,  4)
#gen.add("w2_w2_param", double_t, 0, "Weight neuron 2 to 2", 1.4, -4,   4)

gen.add("weight_self", double_t, 0, "Weight neuron 1 to 1", 1.4, -4,   4)
gen.add("weight_other", double_t, 0, "Weight neuron 1 to 2", 0.7, -4,   4)

gen.add("input1_param", double_t, 0, "Input of the neuron 1", 0, -5,   5)
gen.add("input2_param", double_t, 0, "Input of the neuron 2", 0, -5,   5)

exit(gen.generate(PACKAGE, "dacbot_controller", "two_neuron"))
